src/main/resources/META-INF/rewrite/was-to-liberty.yml

```diff
@@ -134,3 +135,13 @@ recipeList:
       newValue: http://xmlns.jcp.org/xml/ns/javaee
       versionMatcher: 3.1+
       searchAllNamespaces: false
+---
+type: specs.openrewrite.org/v1beta/recipe
+name:  org.openrewrite.maven.liberty.AddOpenLibertyPlugin
+displayName: Add Liberty Maven plugin
+description: This recipe adds  Open liberty plugin to the pom.xml file.
```

Comment:
The description could be improved to say: `This recipe adds the Liberty Maven plugin, which provides several goals for managing a Liberty server and applications.`